ui-grid/ui-grid.d.ts

```diff
@@ -484,6 +484,10 @@ declare module uiGrid {
          */
         appScope?: ng.IScope;
         /**
+        * returns an array of columns in the grid
+        */
+        columns: Array<IGridColumn>;
```

Comment:
@joeskeen It is part of their tutorial (http://ui-grid.info/docs/#/tutorial/308_external_filtering) but I agree that the API documentation can be improved. I'll be submitting a pull request soon.